data/borderlines.json

```diff
@@ -127,6 +127,42 @@
         }
       ]
     },
+    {
+      "name": "Syangja",
+      "latitude": 28.0197,
+      "longitude": 83.8049,
+      "timezone": "Asia/Kathmandu",
+      "officialLanguage": "Nepali",
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 1164,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 1261,
+        "unit": "meters"
+      },
+      "landmarks": [
+        {
+          "name": "Bhalupahad",
+          "latitude": 28.1426,
+          "longitude": 83.8592
+        },
+        {
+          "name": "Sworek Maidan",
+          "latitude": 28.0248,
+          "longitude": 83.7354
+        },
+        {
+          "name": "Bahunthan Hill",
+          "latitude": 27.9896,
+          "longitude": 83.7456
```

Comment:
Can we add a bit more landmarks please?
The most significant ones

Reply:
Please review, more landmarks are added !